<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Anaglyph3DDialog</class>
 <widget class="QDialog" name="Anaglyph3DDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anaglyph 3D generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelLeftImage">
       <property name="text">
        <string>Left Image: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditLeftImage"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLeftImage">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelRightImage">
       <property name="text">
        <string>Right Image:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditRightImage"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRightImage">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelOffset">
       <property name="text">
        <string>Offset:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditOffset"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonGenerate">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
